Dockerfile-base

```diff
@@ -9,56 +9,81 @@ ENV baseDir .
 
 # Install dependencies
 # --------------------
-RUN apt-get update
-RUN apt-get install -y sudo
-RUN apt-get install -y build-essential wget net-tools unzip time imagemagick optipng strace nginx git python zlib1g-dev libpcre3-dev aspell aspell-en aspell-af aspell-am aspell-ar aspell-ar-large aspell-bg aspell-bn aspell-br aspell-ca aspell-cs aspell-cy aspell-da aspell-de aspell-el aspell-eo aspell-es aspell-et aspell-eu-es aspell-fa aspell-fo aspell-fr aspell-ga aspell-gl-minimos aspell-gu aspell-he aspell-hi aspell-hr aspell-hsb aspell-hu aspell-hy aspell-id aspell-is aspell-it aspell-kk aspell-kn aspell-ku aspell-lt aspell-lv aspell-ml aspell-mr aspell-nl aspell-nr aspell-ns  aspell-pa aspell-pl aspell-pt aspell-pt-br aspell-ro aspell-ru aspell-sk aspell-sl aspell-ss aspell-st aspell-sv aspell-tl aspell-tn aspell-ts aspell-uk aspell-uz aspell-xh aspell-zu
+RUN apt-get update \
```

Comment:
The main change in this `RUN` statement is cleaning up `/var/lib/apt/lists`, If that's the case I don't see the benefit to include node installation and nginx file deletion here.

Reply:
The nodejs installation adds a package list and when splitting it into a second RUN stage, we would download the package lists twice.

nginx config files: see below.

Reply:
ok, thanks!